src/Momento.Sdk/PreviewVectorIndexClient.cs

```diff
@@ -7,10 +8,10 @@
 
 namespace Momento.Sdk;
 
-public class PreviewVectorIndexClient: IPreviewVectorIndexClient
+public class PreviewVectorIndexClient : IPreviewVectorIndexClient
 {
     private readonly VectorIndexControlClient controlClient;
-    
+    private readonly VectorIndexDataClient dataClient;
```

Comment:
There is no data client round robin like in the cache client currently, nor are there eager connections or retries. I assumed we could add those in a future pr if there is demand.